### ISSUE TITLE
Nested templates parsing error

### ISSUE THREAD

**Issue by fredericlb** [edgardleal/require-vuejs#20]
Hi,

When using require-vuejs on a vue component with nested templates such as here : 
```
<template>
    <v-app>
        <Header pageName="Facturation"/>
        <sidebar :entity="entity" />
        <section>
            <h2>Historique de prélèvements</h2>
            <v-data-table
                    v-bind:headers="headers"
                    :items="readableArray"
                    hide-actions
                    class="elevation-1"
            >
                <template slot="items" scope="props">
                    <td>{{ props.item.label }}</td>
                    <td class="text-xs-right">{{ props.item.executionDate }}</td>
                    <td class="text-xs-right">{{ props.item.amount }}</td>
                </template>
            </v-data-table>
        </section>
    </v-app>
</template>
```

VueJS issues an error about v-data-table, section, v-app and template not having matching end tag.
It seems that template parsing ends at the first template end tag.

**Comment by edgardleal** [comment]
You are right. The template parser is stopping on first template end tag.
I am working on it.

**Comment by edgardleal** [comment]
What is the version number that you are using ? 
I think this issue was solved on this commit: 0418681f5e6dbf22122f34942e65278428dc1fa5

**Comment by fredericlb** [comment]
My bad, I thought I was up to date but in fact I had quite an old version ! 

Sorry !
